pkg/imageprovider/invalid.go

```diff
@@ -17,3 +24,7 @@ func (ibf ImageBuildInvalid) Unwrap() error {
 func BuildInvalidError(err error) ImageBuildInvalid {
 	return ImageBuildInvalid{err: err}
 }
+
```

Comment:
Can we try to keep the definition and its methods together instead of interleaving them?